app/views/clients/_form.html.haml

```diff
@@ -0,0 +1,11 @@
+= form_for(@client) do |f|
+  - if @client.errors.any?
+    #error_explanation
```

Comment:
I'm seeing this error wrapper a couple of times; maybe we can move this to some partial, and pass the resource as a local variable. Maybe we can call the partial like: `errors_wrapper`? And use it like: `render partial: 'shared/error_wrapper', locals: { resource: @resource }`. What do you think?